src/components/preview/__internal__/preview-placeholder.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import tagComponent from "../../../__internal__/utils/helpers/tags/tags";
+import {
+  StyledPreviewPlaceholder,
+  StyledPreviewPlaceholderProps,
+} from "./preview-placeholder.style";
+
+export interface PreviewPlaceholderProps extends StyledPreviewPlaceholderProps {
+  index: number;
+  /** The number of lines to render. */
+  lines: number;
+  /* Provides more control over when in a loading state. */
+  loading?: boolean;
+}
+
+const PreviewPlaceholder = ({
+  height,
+  index,
+  lines,
+  width,
+  ...props
+}: PreviewPlaceholderProps) => {
+  const isLastLine = lines > 1 && lines === index;
+
+  return (
+    <StyledPreviewPlaceholder
+      key={index}
```

Comment:
I know this was here before but I don't think we need a `key` here at all, as we're not directly rendering `StyledPreviewPlaceholder` in an array of components. The key is added later, inside the actual `Preview` component.

Reply:
right, I have moved the key to the new component but I forgot to remove it here